### PR TITLE
chore: update clean up pipeline

### PR DESCRIPTION
**Pull request by vilit1**
1. change the storage + registry prefixes for init to easily separate the test ran
2. update the clean up pipeline to delete the storage + registry prefixes for init (and just custom prefix if you feel like it)
3. remove keyvault clean up 

---
This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

Thank you for contributing to Azure IoT Operations tooling!

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

Intent for Production

- [ ] It is expected that pull requests made to default or core branches such as `dev` or `main` are of production grade. Corollary to this, any merged contributions to these branches may be deployed in a public release at any given time. By checking this box, you agree and commit to the expected production quality of code.

Basic expectations

- [ ] If introducing new functionality or modified behavior, are they backed by unit and/or integration tests?
- [ ] In the same context as above are command names and their parameter definitions accurate? Do help docs have sufficient content?
- [ ] Have **all** the relevant unit **and** integration tests pass? i.e. `pytest <project root> -vv`. Please provide evidence in the form of a screenshot showing a succesful run of tests locally OR a link to a test pipeline that has been run against the change-set.
- [ ] Have linter checks passed using the `.pylintrc` and `.flake8` rules? Look at the CI scripts for example usage.
- [ ] Have extraneous print or debug statements, commented out code-blocks or code-statements (if any) been removed from the surface area of changes?
- [ ] Have you made an entry in HISTORY.rst which concisely explains your user-facing feature or change?

Azure IoT Operations CLI maintainers reserve the right to enforce any of the outlined expectations.

A PR is considered **ready for review** when all basic expectations have been met (or do not apply).

### CODE REVIEW

**Comment by vilit1** [comment]
Example runs:
new prefix
https://github.com/Azure/azure-iot-ops-cli-extension/actions/runs/11219994335

old prefix
https://github.com/Azure/azure-iot-ops-cli-extension/actions/runs/11187224852

**Comment by vilit1** [comment]
example with custom input
https://github.com/Azure/azure-iot-ops-cli-extension/actions/runs/11220201291
https://github.com/Azure/azure-iot-ops-cli-extension/actions/runs/11220201291/job/31187854277#step:6:1

File: .github/workflows/cluster_cleanup.yml [hunk 1]
```diff
@@ -99,6 +81,22 @@ jobs:
...
+          for resource in $(az resource list -g ${{ env.RESOURCE_GROUP }} --query "[?starts_with(name, '${{ env.REGISTRY_PREFIX }}')].id" -o tsv); do
+            az resource delete -v --id $resource --verbose
+          done
+      - name: Delete storage accounts
+        run: |
+          for resource in $(az resource list -g ${{ env.RESOURCE_GROUP }} --query "[?starts_with(name, '${{ env.STORAGE_PREFIX }}')].id" -o tsv); do
+            az resource delete -v --id $resource --verbose
+          done
+      - name: Delete your custom stuff
+        if: ${{ env.CUSTOM_PREFIX }}
```
Comment (c-ryan-k) [inline .github/workflows/cluster_cleanup.yml:95]: This is dangerous but I understand why it's necessary - perhaps in the future we should add some kind of checking for a "do not delete" tag of sorts?

**Review by c-ryan-k — APPROVED** [review]

File: .github/workflows/cluster_cleanup.yml (hunk 1 above)
Comment (vilit1) [reply .github/workflows/cluster_cleanup.yml:95]: future thing yea but we like to live dangerously

**Pull request merged**